bin/generate-readme

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+transcribe() {
+  local -r project="$(node --print 'require("./package.json").repository.url.replace(/[.]git$/, "").split("/").slice(-2).join("/")')"
```

Comment:
My goal is to make using these scripts in Sanctuary projects (and other projects with the same conventions) as convenient as possible. This means I'm willing to make the scripts less flexible in order to avoid configuration (in some cases default values can be used to let us have our :cake: and eat it too).

We could add a `lint-package-json` script which performs some __package.json__-specific checks in addition to the sanity check performed by `lint-json`. It could assert that `repository.url` is present and that its value is suitable. What do you think of this idea?